### PR TITLE
Follow-up to PR#1942

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1942

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf @tnguyen-rh PTAL. Thanks!

File: install_config/install/deploy_router.adoc [hunk 1]
```diff
@@ -228,21 +228,20 @@ endif::[]
 
 === High Availability
 You can link:../../admin_guide/high_availability.html[set up a highly-available
-router] on your OpenShift cluster using IP failover.
+router] on your [product-title] cluster using IP failover.
```
Comment (bfallonf) [inline install_config/install/deploy_router.adoc]: @ahardin-rh Does it have to be {product-title}? With the squiggly brackets?

**Comment by bfallonf** [comment]
@ahardin-rh One nit. Otherwise, LGTM.

File: install_config/install/deploy_router.adoc [hunk 2]
```diff
@@ -253,79 +252,80 @@ $ oc scale dc/router --replicas=1
...
-
-=== Forcing Route Hostnames to a Custom Routing Subdomain
-
-If an adminstrator wishes to restrict all routes to a specific routing
-subdomain, they can do this by passing the `--force-subdomain` option to the
-`oadm router` command. This forces the router to override any hostnames
-specified in a route and generate one based on the template provided to the
+=== Forcing Route Host Names to a Custom Routing Subdomain
+If an administrator wants to restrict all routes to a specific routing
+subdomain, they can do so by passing the `--force-subdomain` option to the `oadm
```
Comment (tnguyen-rh) [inline install_config/install/deploy_router.adoc]: "If an admin ..., they can do so..." is a bit wordy.

**Comment by tnguyen-rh** [comment]
@ahardin-rh One rew[oa]rding opportunity :laughing: , otherwise LGTM.

File: install_config/install/deploy_router.adoc (hunk 1 above)
Comment (ahardin-rh) [inline install_config/install/deploy_router.adoc]: Woops!  Yes, thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/install/deploy_router.html[Installing -> Deploying a Router]
|Added instructions on link:../install_config/install/deploy_router.html#customizing-the-router-service-ports[customizing router service ports] and updated the usage on how to force routes to a custom routing subdomain.
%
